code/modules/events/blowout.dm

```diff
@@ -88,6 +89,7 @@
 					break
 				if (!A.permarads)
 					A.irradiated = 0
+				A.icon_state = ""
```

Comment:
```suggestion
				A.icon_state = null
```

Reply:
BTW this was copied from battlestorm so it will be wrong there too